tests/karate/src/test/java/be/LAO/create.feature

```diff
@@ -120,3 +125,22 @@ Feature: Create a pop LAO
     And json answer = frontend.getBackendResponse(JSON.stringify(laoCreateRequest))
     Then match answer contains ACCESS_DENIED
     And match frontend.receiveNoMoreResponses() == true
+
+
+  Scenario: Create Lao with different public key from the organizer should fail with error response
+    Given def laoCreateRequest =
+      """
+        {
+          "object": "lao",
+          "action": "create",
+          "id": '#(laoId)',
+          "name": "LAO",
```

Comment:
Yes, the idea would be to dynamically compute it, ideally without having to give a creation time as a parameter of the lao id generation function (just take the system time). However, this might not work if I need to pass the creation time again here in the json. But maybe I can find a way to access it here.